### PR TITLE
improved welcome email UI

### PR DESCRIPTION
**Pull request by ALI-KHALED-13**

### CODE REVIEW

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 1]
```diff
@@ -1,47 +1,63 @@
...
+        <li>Track your activity from your personal dashboard</li>
+      </ul>
+      <p>
+        Additionally, if you own a product/bussiness and want to build a bridge of communication with your customers you can:
+      </p>
+      <ul>
+        <li>Initiate your own product feedback stream</li>
+        <li>Take customer suggestions and start discussions in the comments</li>
+        <li>Promote a feedback status from a mere suggestion to <b>Planned</b>, <b>In-progress</b> or <b>Live</b> as you actulize said suggestion</li>
+        <li>Add your Business's team as <b>contributers</b> to give them the authoirty to promote suggestions too</li>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:53]: "contributOrs"

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 2]
```diff
@@ -1,47 +1,63 @@
...
+        <li>Comment on other People's feedback and have discussions</li>
+        <li>Track your activity from your personal dashboard</li>
+      </ul>
+      <p>
+        Additionally, if you own a product/bussiness and want to build a bridge of communication with your customers you can:
+      </p>
+      <ul>
+        <li>Initiate your own product feedback stream</li>
+        <li>Take customer suggestions and start discussions in the comments</li>
+        <li>Promote a feedback status from a mere suggestion to <b>Planned</b>, <b>In-progress</b> or <b>Live</b> as you actulize said suggestion</li>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:52]: "actuAlize"

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 3]
```diff
@@ -1,47 +1,63 @@
...
+      <h3>Hello <b>{{name}}!</b> We're thrilled you joined our hub!.</h3>
+      <img src="cid:welcome_cats" alt="Welcome Aboard"/>
+      <p> Super excited to see your future contributions 🤩</p>
+    </section>
+    
+    <article id="getStarted">
+      <p>on Feedback-Hub you can:</p>
+      <ul>
+        <li>Browse different product feedback streams</li>
+        <li>give feedback to a product of interest</li>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:41]: Capital letter

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 4]
```diff
@@ -1,47 +1,63 @@
...
+  <body>
+
+    <section id="welcome">
+      <h3>Hello <b>{{name}}!</b> We're thrilled you joined our hub!.</h3>
+      <img src="cid:welcome_cats" alt="Welcome Aboard"/>
+      <p> Super excited to see your future contributions 🤩</p>
+    </section>
+    
+    <article id="getStarted">
+      <p>on Feedback-Hub you can:</p>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:38]: Capital letter

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 5]
```diff
@@ -1,47 +1,63 @@
...
+        margin: 0 auto;
+      }
+      li {margin: 0.5rem;}
+      li::after {content: ".";}
+    </style>
+  </head>
+  <body>
+
+    <section id="welcome">
+      <h3>Hello <b>{{name}}!</b> We're thrilled you joined our hub!.</h3>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:32]: Extra period after the exclamation mark

File: api/utils/sendEmail/tpls/welcome.handlebars [hunk 6]
```diff
@@ -1,47 +1,63 @@
...
+      <p> Super excited to see your future contributions 🤩</p>
+    </section>
+    
+    <article id="getStarted">
+      <p>on Feedback-Hub you can:</p>
+      <ul>
+        <li>Browse different product feedback streams</li>
+        <li>give feedback to a product of interest</li>
+        <li>Upvote other people's feedback to push it to the top and increase its chances to turn into reality</li>
+        <li>Comment on other People's feedback and have discussions</li>
```
Comment (Elena-Erganyan) [inline api/utils/sendEmail/tpls/welcome.handlebars:43]: "people's" (should start with a small letter)

**Review by Elena-Erganyan — COMMENTED** [review]
Nicely done, looks so cute 😍 

I even used the same pic for the ActivationMessage page, was good to see it here as well 😆 (changed it, not to repeat it everywhere + it fits perfectly here).

I left some comments, they are regarding typos and English grammar a bit :)

File: api/utils/sendEmail/tpls/welcome.handlebars (hunk 5 above)
Comment (ALI-KHALED-13) [reply api/utils/sendEmail/tpls/welcome.handlebars:32]: no need for that

File: api/utils/sendEmail/tpls/welcome.handlebars (hunk 1 above)
[the remark this answers is not in the document]
Comment (ALI-KHALED-13) [reply api/utils/sendEmail/tpls/welcome.handlebars:53]: I'll leave as is

**Review by Elena-Erganyan — APPROVED** [review]

**Pull request merged**